core/persistence-api/src/main/java/org/apache/syncope/core/persistence/api/dao/AuditDAO.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.syncope.core.persistence.api.dao;
+
+import org.apache.syncope.core.persistence.api.dao.search.OrderByClause;
+import org.apache.syncope.core.persistence.api.dao.search.SearchCond;
+import org.apache.syncope.core.persistence.api.entity.AuditEntry;
+
+import java.util.List;
+import java.util.Set;
+
+public interface AuditDAO<T extends AuditEntry> {
+    AuditEntry findByUsername(String who);
```

Comment:
Searching for Audit entries should not be limited to users: maybe `findByEntityKey()` would make sense as we are planning to lookup entries related to Users, Groups and AnyObjects (for the moment) but possibly also for Resource and Connnector Instances in the near future.